### PR TITLE
Update the OPAM packages

### LINKED ISSUE #159: Update OPAM packages
The OPAM packages must be updated for the release. In particular, it would be nice for them to have some descriptions.

### PR DESCRIPTION
**Pull request by shym**
This is incomplete at the moment.

Closes #159

### CODE REVIEW

**Comment by shym** [comment]
This is now in a good shape, I think, it can be reviewed.
As some fields are just moved around, `--color-moved=dimmed-zebra` (for instance) can be helpful to review.

File: ortac-core.opam [hunk 1]
```diff
@@ -1,16 +1,35 @@
 opam-version: "2.0"
-synopsis: "OCaml Runtime Assertion Checking based on Gospel"
-description: "OCaml Runtime Assertion Checking based on Gospel"
-maintainer: ["Clément Pascutto <clement@pascutto.fr>"]
+synopsis:
+  "Ortac (OCaml Runtime Assertion Checking based on Gospel) core tool and library"
```
Comment (n-osborne) [inline ortac-core.opam:3]: That's the only place where "based on Gospel" is present inside the parenthesis.
I would remove them as this is not part of the acronym.

**Review by n-osborne — APPROVED** [review]
Great, thank you!

I just had one tiny remark.

Also, this can wait that the documentation has  been published, or we can add the URLs for the documentation in another PR.

**Comment by shym** [comment]
Good points!
I added the link to the documentation for the 3 packages to be released. And took the “based on Gospel” out of its parentheses, I agree it makes more sense that way. I also like the idea that the synopsis of `ortac-core` really mentions Gospel. I wondered whether the other packages should also have it appear in their synopsis, I wasn’t so sure.

**Comment by shym** [comment]
After a last complete run, I removed the CI part that torture-test the installation of the opam packages, as that’s of little interest besides preparing for an upcoming release. I think this is ready for release now.

**Pull request merged**